### PR TITLE
Update GTM and add cookie policy

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update GTM and add cookie policy
- Removed the optimise code as it isn't being used

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8041
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the cookie popup works, it in the footer and the GTM code is `GTM-K9KCMZ`

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3269

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
   "license": "LGPL v3",
   "scripts": {
-    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
+    "clean": "rm -rf node_modules yarn-error.log css static/css static/js/build *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
     "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
     "build": "yarn run build-css && yarn run build-js",
     "build-css": "node-sass --include-path node_modules static/sass --source-map true --output-style compressed --output static/css && postcss --map false --use autoprefixer --replace 'static/css/**/*.css'",
-    "build-js": "mkdir -p static/js/build/global-nav && cp node_modules/@canonical/global-nav/dist/global-nav.js static/js/build/global-nav",
+    "build-js": "yarn run build-global-nav && yarn run build-cookie-policy",
+    "build-global-nav": "mkdir -p mkdir -p static/js/build/global-nav && cp node_modules/@canonical/global-nav/dist/global-nav.js static/js/build/global-nav",
```
Comment (anthonydillon) [inline package.json:10]: You have double `mkdir -p` here

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Issue in the npm script. Why not push to the existing PR?

**Comment by pmahnke** [comment]
@anthonydillon we can close this, I didn't see you had a PR, strange, I did check

**Comment by anthonydillon** [comment]
No your PR has the GTM code updates which mine so it's fine.

**Review by anthonydillon — APPROVED** [review]
Nice catch on the clean command :+1:

**Pull request merged**